src/igvfd/audit/construct_library_set.py

```diff
@@ -118,3 +118,34 @@ def audit_construct_library_set_guide_design(value, system):
             detail = (f'Construct library set {audit_link(path_to_text(value["@id"]),value["@id"])} has no '
                       f'`integrated_content_files`.')
             yield AuditFailure('missing guide RNA sequences', f'{detail} {description}', level='NOT_COMPLIANT')
+
+
+@audit_checker('ConstructLibrarySet', frame='object')
+def audit_construct_library_set_orf_gene(value, system):
+    '''
+    [
+        {
+            "audit_description": "Genes listed in the library are expected to match the open read frame gene.",
+            "audit_category": "inconsistent gene",
```

Comment:
I think `inconsistent genes` is more consistent with category naming guidelines.

Reply:
done